static/openapi/est-spec.json

```diff
@@ -0,0 +1,1543 @@
+              "type": "string",
+              "format": "date-time"
+            },
+            "description": "Optional. Filter results up to a specified date and time in UTC (e.g. 2022-01-01T00:00:00Z)."
+          },
+          {
+            "in": "query",
+            "name": "limit",
+            "schema": {
+              "type": "integer",
+              "minimum": 1,
+              "maximum": 1000
+            },
+            "description": "Optional. Limit the number of results (positive integer, default is 20)."
+          },
+          {
+            "in": "query",
+            "name": "offset",
+            "schema": {
+              "type": "integer",
+              "minimum": 0
+            },
+            "description": "Optional. Offset the results by a non-negative integer (default is 0)."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns the change history for PhotoniqEST based on the provided query parameters."
+          },
+          "400": {
+            "description": "Bad Request. Invalid query parameter."
+          },
+          "500": {
+            "description": "Internal Server Error. Error getting change history."
+          }
+        }
+      }
+    },
+    "/api/est/v1/components": {
+      "get": {
+        "tags": [
+          "Components"
+        ],
+        "description": "Get all components supported by PhotoniqEST service",
+        "parameters": [
+          {
+            "in": "query",
+            "name": "category",
+            "schema": {
+              "type": "string"
+            },
+            "description": "Filter components by category (optional)"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns list of components supported by PhotoniqEST"
+          },
+          "500": {
+            "description": "Internal Server Error. Error getting list of components"
+          }
+        }
+      }
+    },
+    "/api/est/v1/components/{component}": {
+      "get": {
+        "tags": [
+          "Components"
+        ],
+        "description": "Get Component metadata by component name",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "component",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "Name of the component"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns Component metadata"
+          },
+          "404": {
+            "description": "Component not found"
+          },
+          "500": {
+            "description": "Internal Server Error. Failed to retrieve the component"
+          }
+        }
+      }
+    },
+    "/api/est/v1/components/settings": {
+      "post": {
+        "tags": [
+          "Settings"
+        ],
+        "description": "Create PhotoniqEST component setting",
```

Comment:
@diana-macrometa do you think we should use PhotoniqEST all these places.

Would "Create component setting" work?

None of our other APIs are referencing the service in all these descriptions like this swagger apis.

Reply:
Yes, your edit is better. Let's do that to keep consistency.